docs/writing-tests/test-runner.md

```diff
@@ -0,0 +1,293 @@
+---
+title: 'Test runner'
+---
+
+Storybook test runner turns all of your stories into executable tests. It is powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/).
+
+- For those [without a play function](../writing-stories/introduction.md): it verifies whether the story renders without any errors.
+- For those [with a play function](../writing-stories/play-function.md): it also checks for errors in the play function and all assertions passed.
+
+These tests run in a live browser and can be executed via the [command line](#cli-options) or your [CI server](#set-up-ci-to-run-tests).
+
+## Setup
+
+The test runner is a standalone, framework-agnostic addon that runs parallel to your Storybook. You will need to take some additional steps to set it up properly. Detailed below is our recommendation to configure and execute it.
```

Comment:
> addon

Are we using that term more generally than what gets registered under `main.js`'s `addons`?

Perhaps "library" or "package" would be better?

Reply:
Good catch. I initially had utility. Would that work better?

Reply:
Rephrased it to utility. Marking this as solved.